project/backend/office-service/docker-compose.yml

```
@@ -0,0 +1,30 @@
+      # Connects to 'metadocs' DB on the host via Docker Desktop's special DNS name
+      - DB_URL=jdbc:mysql://host.docker.internal:3306/metadocs # <<< USE THIS VALUE
+      - DB_USERNAME=root                                     # <<< PROVIDE USERNAME
+      - DB_PASSWORD=1234                                     # <<< PROVIDE PASSWORD (Use secrets!)
+
+    networks:
+      - monitoring_net
+      - default
+
+networks:
+  monitoring_net:
+    external: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)